src/openrct2/paint/tile_element/Paint.Path.cpp

```diff
@@ -1373,3 +1386,30 @@ void PathPaintPoleSupport(
         PaintUtilSetSegmentSupportHeight(session, SEGMENT_C8, 0xFFFF, 0);
     }
 }
+
+void PathPaintOnlyAddition(
+    PaintSession& session, const PathElement& pathElement, int16_t height, const FootpathPaintInfo& pathPaintInfo,
+    ImageId imageTemplate, ImageId sceneryImageTemplate)
+{
+    PROFILED_FUNCTION();
+
+    // Rol edges around rotation
+    uint8_t edges = ((pathElement.GetEdges() << session.CurrentRotation) & 0xF)
+        | (((pathElement.GetEdges()) << session.CurrentRotation) >> 4);
+
+    uint8_t corners = (((pathElement.GetCorners()) << session.CurrentRotation) & 0xF)
+        | (((pathElement.GetCorners()) << session.CurrentRotation) >> 4);
+
+    uint16_t edi = edges | (corners << 4);
```

Comment:
can you just call it edgesAndCorners can't stand adding a register name when this wasn't reverse engineered code and we are meant to be getting rid of them.

Reply:
It's copy/pasted from the other one but I can change it.